### ISSUE TITLE
Request New type of device

### ISSUE THREAD

**Issue by betamax2021** [AMoo-Miki/homebridge-tuya-lan#67]
Would it be possible to add a device type for RGBWW led strips?

**Comment by AMoo-Miki** [comment]
@betamax2021, absolutely. Do you have this device? Please follow [these instruction](https://github.com/AMoo-Miki/homebridge-tuya-lan/wiki/Supported-Devices#adding-supported-for-new-devices) to get the signature and post it here.
